java/core/src/java/org/apache/orc/OrcConf.java

```diff
@@ -164,8 +164,8 @@
       "testing.  Setting this too low may negatively affect performance."),
   OVERWRITE_OUTPUT_FILE("orc.overwrite.output.file", "orc.overwrite.output.file", false,
     "A boolean flag to enable overwriting of the output file if it already exists.\n"),
-  IS_SCHEMA_EVOLUTION_CASE_SENSITIVE("orc.schema.evolution.case.sensitive",
-      "orc.schema.evolution.case.sensitive", true,
+  IS_SCHEMA_EVOLUTION_CASE_AWARE("orc.schema.evolution.case.aware",
+      "orc.schema.evolution.case.aware", true,
```

Comment:
I'm not sure if this change will affect `hiveConf`

Reply:
So would it be better to change `isSchemaEvolutionCaseAware` to `isSchemaEvolutionCaseSensitive`?

Reply:
Yes, But there are many changes in this `isSchemaEvolutionCaseAware` to `isSchemaEvolutionCaseSensitive` and the default value also needs to be modified to be consistent with currently one.

Reply:
BTW, Do we think the default value should be insensitive or aware, At present, it is aware